src/server/log.ml

```diff
@@ -383,6 +388,7 @@ let log =
 
 
 let set_up_exception_hook () =
+  (* TODO Is there an Eio equivalent?
```

Comment:
I think no, because to my understanding every fiber in Eio propagates its exceptions upwards to its parent fibers and eventually up to the root fiber.